decidim/decidim
# Bump shakapacker from 6.6.0 to 7.1.0

#### :tophat: What? Why?
This PR updates Shakapacker to the latest available version. IT cleans out unused files

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to https://github.com/shakacode/shakapacker/issues/365

#### Testing
1. Create a test application 
2. Try to run the spec `bundle exec rspec spec/system/editor_spec.rb -e "with full toolbar controls image"`
3. See there is an error
4. Apply the patch 
5. repeat 1 & 2 
6. See there is no error

:hearts: Thank you!
